### PR TITLE
Target cloud during tests unless other endpoint is specified

### PR DESCRIPTION
**Pull request by erickpintor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,11 +80,9 @@ GitHub pull requests are very welcome.
 
 ### Driver Development
 
-Run `go get -t ./...` in order to install project's dependencies. Run tests with
-`go test ./...`.
+Run `go get -t ./...` in order to install project's dependencies.
 
-Tests will attempt to connect to `localhost` by default. Check the default test
-parameters at `faunadb_test.go`.
+Run tests with `FAUNA_SECRET="your-could-secret" go test ./...`.
```
Comment (ashfire908) [inline README.md]: `cloud`?

Comment (erickpintor) [inline README.md]: Thanks

File: README.md [hunk 2]
```diff
@@ -80,11 +80,9 @@ GitHub pull requests are very welcome.
 
 ### Driver Development
 
-Run `go get -t ./...` in order to install project's dependencies. Run tests with
-`go test ./...`.
+Run `go get -t ./...` in order to install project's dependencies.
 
-Tests will attempt to connect to `localhost` by default. Check the default test
-parameters at `faunadb_test.go`.
+Run tests with `FAUNA_SECRET="your-cloud-secret" go test ./...`.
```
Comment (eaceaser) [inline README.md]: the env far is FAUNA_ROOT_KEY below

**Comment by erickpintor** [comment]
Opps. Fixed.

**Comment by marrony** [comment]
lgtm

**Pull request merged**